cpu/cc2538/dev/cc2538-ccm-star.c

```diff
@@ -75,49 +75,18 @@ restore_crypto(uint8_t enabled)
 static void
 set_key(const uint8_t *key)
 {
-  cc2538_aes_128_driver.set_key(key);
+  ccm_star_driver.set_key(key);
```

Comment:
I do not have a concise example, but [this](https://github.com/kkrentz/contiki/tree/potr/examples/secmac/tests/potr) one. It crashes after sending 100-10000 frames when using the old implementation.